tests/regression/37-congruence/01-simple.c

```diff
@@ -0,0 +1,34 @@
+//PARAM: --enable ana.int.congruence  --enable ana.int.congruence_no_overflow --disable ana.int.def_exc --disable ana.int.enums
```

Comment:
Please add a comment to make it clear that this example comes straight from Granger's paper, in order to give proper credit to him.